tests/snapshots/tests.contrib.graphql.test_graphql.test_graphql_error_v2_promise.json

```diff
@@ -0,0 +1,57 @@
+[[
+  {
+    "name": "graphql.query",
+    "service": "graphql",
+    "resource": "{ invalid_schema }",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "web",
+    "meta": {
```

Comment:
`graphql.graphql()` returns  promise. Although an error occurred in this span this is not known until the promise resolves. Unlike the `test_graphql_error.json` snapshot this error is only marked in the validate span.

Reply:
This was fixed by patching `graphql.execute_graphql` instead of `graphql.graphql` in graphql-core-2